### ISSUE TITLE
Theming

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/deltachat-desktop#889]
#457 #409 
Meta Issue to track the state of implementing theming:

- [x] For the time being - expose sass vars also to the theming #858
- [ ] create theming vars for the missing things [~87% completed]
- [x] Think about what color vars make sense to be grouped / rename the signal ones (signal-blue is super non-telling)
- [ ] Pull theme from file/ save in file not in local storage
- [ ] Change Theme setting in settings
- [ ] Detect System Theme (Dark/Light)
- [ ] theme new buttons from background selection
- [ ] make buttons adapt better to darkmode
- [x] make media view bg themable (replace sc)
- [ ] .bp3-navbar.bp3-fixed-top
- [ ] make attachments overview themable (replace sc)
- [ ] input fields in settings (dark & amoled theme)
- [ ] unblock contact screen
- [ ] theme audio messages
- [ ] add padding to bottom of map
- [x] Detect background color and change welcome image to maintain contrast? (only really important for amoled theme)
- [ ] scrollbar

**Comment by MIntrovrt** [comment]
Having some kind of functionality that imports themes from the ~/.Xresources file would be nice to have

**Comment by Simon-Laux** [comment]
This could also help for finding out which theme is active: https://electronjs.org/docs/api/system-preferences#systempreferences

**Comment by MIntrovrt** [comment]
another idea: theme map

**Comment by Simon-Laux** [comment]
> another idea: theme map

I wouldn't spend much time on the map. its experimental and will probably change in the future.

**Comment by pabzm** [comment]
Theming is a feature request or improvement, which is not high on our roadmap. Can we please separate this into bugs that need fixing, and a wishlist that we move to Project resurrection?

**Comment by Simon-Laux** [comment]
It's not useful to have 10 bugs about each little minor thing that needs theming work.

**Comment by pabzm** [comment]
I think it is. Concrete bugs with proper descriptions.

Everything that is not a concrete bug is (part of) a feature request, which shall not stay open in this tracker (that's our policy, not my personal idea). Theming is a nice idea, but will have to wait — in the forum.

Additionally this had not seen any activity since 9 weeks, which alone would be a reason to maybe close it.

How do you suggest to continue?

**Comment by Simon-Laux** [comment]
I'm assigned to it so I suggest to leave it open. I will try to merge @MIntrovrt second branch which fixes some of the issues soon.

**Comment by Simon-Laux** [comment]
the other option would be project resurrection
